packages/remix-server-runtime/server.ts

```diff
@@ -24,14 +24,20 @@ export interface RequestHandler {
   (request: Request, loadContext?: AppLoadContext): Promise<Response>;
 }
 
-/**
- * Creates a function that serves HTTP requests.
- */
-export function createRequestHandler(
+export type CreateRequestHandlerFunction = (
   build: ServerBuild,
   platform: ServerPlatform,
```

Comment:
> I don't think we're really using this for anything. 

If by "this" you mean the `platform` arg, then sure, I can remove it.

Reply:
Yep, that's what I meant. I think we can refactor it out of the codebase with this work. It's ok if the function signature changes here since the remix-server-runtime interface isn't yet considered public or documented anywhere.